static/js/util/profile_edit.js

```diff
@@ -162,6 +164,9 @@ export function boundSelectField(keySet, label, options) {
     _.set(clone, editKeySet, undefined);
 
     updateProfile(clone);
+    if (onChange !== undefined) {
```

Comment:
maybe `_.isFunction` instead? if we get some unexpected value passed in we could get a runtime error (`Uncaught TypeError: foobar is not a function`).